src/AnsiUtils.ps1

```diff
@@ -47,6 +47,12 @@ function Test-VirtualTerminalSequece([psobject[]]$Object, [switch]$Force) {
 }
 
 function Get-VirtualTerminalSequence ($color, [int]$offset = 0) {
+    # Don't output ANSI escape sequences if the `$color` parameter is `$null`,
```

Comment:
Minor nit - probably don't need the markdown syntax in a comment.

Reply:
Yeah, but I feel like
```pwsh
# Don't output ANSI escape sequences if the `$color` parameter is `$null`,
```
looks better than
```pwsh
# Don't output ANSI escape sequences if the $color parameter is $null,
```